fidesctl/src/fidesctl/core/export_helpers.py

```diff
@@ -301,23 +306,48 @@ def union_data_categories_in_joined_dataframe(joined_df: pd.DataFrame) -> pd.Dat
     """
 
     # isolate the system data categories into a new dataframe and create a common column
+    joined_df = joined_df.drop(
+        [
+            "system.description",
+            "system.privacy_declaration.name",
+            "dataset.description",
+        ],
+        axis=1,
+    )
     systems_categories_df = joined_df.drop(
-        ["dataset.data_categories", "dataset.retention"], axis=1
-    ).drop_duplicates()
+        [
+            "dataset.data_categories",
+            "dataset.retention",
+            "dataset.data_qualifier",
+        ],
+        axis=1,
+    )
+    systems_categories_df["dataset.name"] = "N/A"
     systems_categories_df["unioned_data_categories"] = systems_categories_df[
         "system.privacy_declaration.data_categories"
     ]
+    systems_categories_df["unioned_data_qualifiers"] = systems_categories_df[
+        "system.privacy_declaration.data_qualifier"
+    ]
     systems_categories_df["dataset.retention"] = "N/A"
+    systems_categories_df["dataset.data_qualifier"] = "N/A"
```

Comment:
I think these reassignments can likely be grouped, but need to verify

Reply:
I think this is more readable as is, so planning on leaving it unless someone has a different opinion